packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -123,6 +123,9 @@ contract SystemConfig is ProxyAdminOwnedBase, OwnableUpgradeable, Reinitializabl
     /// @notice The EIP-1559 elasticity multiplier.
     uint32 public eip1559Elasticity;
 
+    /// @notice The min base fee log2.
+    uint8 public minBaseFeeLog2;
+
```

Comment:
Any storage variables we add have to come after the pre-existing storage variables for upgrades to work correctly. As written, upgrading a chain will set that chain's `minBaseFeeLog2` to the existing value of `operatorFeeScalar` and the following three storage variables will have cascading unintentional changes. (I think `l2ChainId` and `superchainConfig` aren't live anywhere yet, but we should still add our variable(s) after those.)